src/test/python/jointGenerator/rockInput.py

```diff
@@ -0,0 +1,116 @@
+#!/usr/bin/python
+import numpy as np
+import sys
+from Joint import Joint
```

Comment:
How about making the file `joint.py` instead of `Joint.py`? Then we can at least do `from joint import Joint` to eliminate some of the ambiguity between the module and class.